src/core/load-fragment.c

```diff
@@ -6494,3 +6494,33 @@ int config_parse_watchdog_sec(
 
         return 0;
 }
+
+int config_parse_tty_size(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        unsigned *sz = data;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        if (isempty(rvalue))
+                *sz = UINT_MAX;
+        else {
+                r = safe_atou(rvalue, sz);
+                if (r < 0)
+                        return r;
```

Comment:
should log with log_syntax() 

(or alternatively, just chain call config_parse_unsigned() here which logs already)